samply/src/windows/profile_context.rs

```diff
@@ -535,6 +551,13 @@ impl ProfileContext {
             reference_raw: timestamp_raw,
             raw_to_ns_factor: 1000 * 1000 * 1000 / perf_freq,
         };
+
+        self.time_range = self.time_range_us.map(|(start, end)| {
+            (
+                self.timestamp_converter.convert_us(start),
+                self.timestamp_converter.convert_us(end),
```

Comment:
Doesn't this treat the range as being relative to the QPC origin, rather than relative to profiling start?

Reply:
I would have thought you could just use `Timestamp::from_nanos_since_reference` in the constructor. Or do you want to be able to specify ranges relative to the QPC origin?

Reply:
Oops, it should not be `convert_us`, since that's a duration; it should be from_.._since reference. But I can't do it in the ctor -- the timestamp_converter isn't fully set up at that point. If I want to keep these as Timestamps, I need them to have the same origin as the rest of the times.

It also is kind of wonky and unfortunate that ProfileContext's methods are raw timestamps instead of `Timestamp`, since they always come from the event timestamp. Any objections to me fixing?

Reply:
Oh wait, right. I see what you mean. I had timestamp origin confused in my head.

Reply:
> It also is kind of wonky and unfortunate that ProfileContext's methods are raw timestamps instead of Timestamp, since they always come from the event timestamp. Any objections to me fixing?

No objections, unless the fix comes with a RefCell